src/vm/clrex.cpp

```diff
@@ -1850,7 +1845,7 @@ OBJECTREF EEFileLoadException::CreateThrowable()
     ARG_SLOT args[] = {
         ObjToArgSlot(gc.pNewException),
         ObjToArgSlot(gc.pNewFileString),
-        ObjToArgSlot(gc.pFusLogString),
+        NULL,
```

Comment:
If you wanted to, you can go one step further and delete the `string? fusionLog` argument from the constructor in FileLoadException.CoreCLR.cs, BadImageFormatException.CoreCLR.cs and FileNotFoundException.CoreCLR.cs and not pass this NULL here at all. (It would also need changing `gsig_IM_Str_Str_Int_RetVoid` a few lines above to `gsig_IM_Str_Int_RetVoid`.)